### DISCUSSION TITLE
Validation

### DISCUSSION THREAD

**Discussion by maretrust** [dj-fiorex/angular2-smart-table#104] [General]
HI, 

thank you for your maintain the project !

Exist the feature for validate the data input in inline edit, on confirm create and confirm modify ?

Thank you for you response !

**Comment by uap-universe** [comment]
Sorry for the late response - I do not look frequently into the Discussions area and I don't get notifications from Github when new threads are created.

Of course you can. You can start with the [confirm action example](https://dj-fiorex.github.io/angular2-smart-table/examples/various) and instead of showing a confirmation dialog, you can perform your validation and maybe show a error dialog or display an error message on your page whatever suits your requirements.
